app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  validates :user, presence: true
+  validates :post, presence: true
```

Comment:
#### Missing Validation
- [ ] `body` - Should not be able to receive an empty value. Missing Validation